plugins/jump/README.md

```diff
@@ -2,10 +2,25 @@
 
 The jump plugin enables jump on bash.
 
-To use it, install
-[jump](https://github.com/gsamokovarov/jump?tab=readme-ov-file#installation)
-and add jump to the plugins array of your bashrc file:
+Jump integrates with your shell and learns about your navigational habits by keeping track of the directories you visit. It gives you the most visited directory for the shortest search term you type.
 
-```bash
-plugins=(... jump)
-```
+## Installation
+
+1. [Install jump](https://github.com/gsamokovarov/jump?tab=readme-ov-file#installation)
+
+2. Enable the plugin by adding it to the plugins array of your bashrc file:
+
+    ```bash
+    plugins=(... jump)
+    ```
+
+3. Enable the completions by adding it to the completions array of bashrc file:
+
+    ```bash
+    completions=(... jump)
+    ```
+
```

Comment:
This explanation appears to imply that both the `jump` plugin and the `jump` completion are mandatory, which is not the case. Could you elaborate on the wording so that each of the plugin and completion can be enabled at the user's discretion?